Interface/AddOns/nMainbar/config.lua

```diff
@@ -17,10 +17,13 @@ nMainbar.Config = {
 
         hotkeyFontsize = 18,
         hotkeyFont = 'Interface\\AddOns\\nMainbar\\media\\font.ttf',
+
+        petHotKeyFontsize = 14,
+        petHotKeyFont = 'Interface\\AddOns\\nMainbar\\media\\font.ttf',
```

Comment:
I just swapped it out for hotkeyFont. I didn't feel it was really needed either but I wanted to leave the option open.